pkg/lib/bundle/generate.go

```diff
@@ -350,6 +357,22 @@ func GenerateAnnotations(mediaType, manifests, metadata, packageName, channels,
 	return afile, nil
 }
 
+func GetAnnotations(content []byte) (Annotations, error) {
+	var annotationsFile AnnotationMetadata
+	err := yaml.Unmarshal(content, &annotationsFile)
+	if err != nil || annotationsFile.Annotations == nil {
+		return Annotations{}, fmt.Errorf("failed to decode annotations file")
+	}
+	return Annotations{
```

Comment:
Similar comment here, why not just use the native type now that we're defined it?